jms/1.7/modules/ROOT/pages/jms-publish-consume.adoc

```diff
@@ -1,82 +1,106 @@
+. In the *Message* section, set the *Reply To* field to *Edit inline*.
+. Set the *Destination Name* field to `replyToDestination`.
+. Scroll down to the *Consume Configuration* section and set the *Maximum wait* field to `20`,
+. Set the *Maximum wait unit* to `SECONDS`.
+
+.Known Queue configuration
+image::jms-publishconsume-config-2.png[In the Publish consume configuration screen, set the Reply To field to The JMSReplyTo header information of the Destination where the message is replied to]
+
+In the XML editor, the `destination` and `<jms:reply-to>` configuration looks like this:
 
 [source,xml,linenums]
 ----
 <jms:publish-consume config-ref="config" destination="targetDestination">
     <jms:message>
         <jms:reply-to destination="replyToDestination"/>
     </jms:message>
+			<jms:consume-configuration maximumWait="20" maximumWaitUnit="SECONDS" />
 </jms:publish-consume>
 ----
 
-Now the operation publishes a message to the `targetDestination` with the
-current `payload` as its body and the `JMSReplyTo` header configured with
-the `replyToDestination`.
-Then, it automatically tries to consume the response from the
-`replyToDestination` queue until a message is consumed or the `maximumWait` is
-reached, in which case it fails with a `JMS:TIMEOUT` error.
 
-== Request Reply Patterns
 
-As of JMS Connector 1.6.0:
+== Configure Request and Reply Patterns
+
+In JMS, exists two well-known request and reply patterns to understand how request and reply messages are correlated. In the *Request-Reply Pattern* field of the operation you can configure the *CORRELATION_ID (Default)* and *MESSAGE_ID* patterns. Both ensure that an incoming message is the proper reply of an outgoing message.
+You can also configure the field to *NONE* to not use any of the patterns, and the consumer tries to consume from the reply queue without using any selector to correlate messages. This option is useful when working with temporal queues that ensure that the queue has only one message.
+
+=== Configure the Correlation ID Pattern
 
-In JMS, there exists two well know Request-Reply patterns for how request and reply messages
-are correlated. This ensures that an incoming message is the proper reply
-of an outgoing message.
+The *CORRELATION_ID (Default)* pattern implies that the operation after publishing a message to the destination queue, consumes a message using a selector that expects a message with the same correlation id to be used to publish the request message.
 
-=== Correlation ID Pattern
+In the following example, in the first flow you configure the *Publish consume* operation to publish a message to the `targetDestination` with the current `payload` as its body and the JMSReplyTo header configured to the known queue setting the *Reply To* field to `replyToDestination`. Also, the operation has configured the *Request-Reply Pattern* field to *CORRELATION_ID (Default)*. +
```

Comment:
This is just a suggestion, but I always feel like it's easier to understand what's going on if you turn this into an ul, like:
In the following example, in the first flow, you configure the *Publish consume operation* to publish a message to the `targetDestination` with the:

* Current `payload` as its body
* `JMSReplyTo` header configured to the known queue
* *Reply To* field set to `replyToDestination`
* *Request-Reply Pattern* field set to *CORRELATION_ID (Default)*